test/cqlpy/test_select_from_mutation_fragments.py

```diff
@@ -12,11 +12,10 @@
 import glob
 import json
 import os
-import nodetool
+from . import nodetool, util
```

Comment:
Here too, please leave two separate lines, not one line. "nodetool" and "util" are two separate files, just like "glob" and "json" above.